src/application/models/v2/Appointments_model_v2.php

```diff
@@ -317,12 +318,31 @@ public function getAllAppointmentBy($service, $aggregates = FALSE, $startDate, $
             $appointments = $this->db->get_where('ea_appointments', $condition)->result_array();
         }
 
+        $maleBookingCounter = 0;
+        $femaleBookingCounter = 0;
+        $undefinedGenderBooking = 0;
         if ($aggregates) {
             foreach ($appointments as &$appointment) {
                 $appointment = $this->get_aggregates($appointment);
+                $gender = $appointment['customer']['gender'];
+                switch(strtoupper($gender)){
```

Comment:
Could we use group by to get the result. 'cause right now the value of gender if not M and F,
we have to change the relate api

Reply:
Hi please refresh the page or see in file changed tab, this is old solution and I already changed it 10 years ago. @jasminecrazy